### PR TITLE
Add a helper function to simplify adding the import hook

### PR DESCRIPTION
**Pull request by theonewolf**
Thanks to @pradyunsg for this function.

### CODE REVIEW

File: pyce/_imports.py [hunk 1]
```diff
@@ -164,3 +164,17 @@ def find_spec(cls, fullname: str, path: Optional[List[str]] = None,
             if sorocospec is not None:
                 break
         return sorocospec
+
+
+def add_import_hook(keys={}):
```
Comment (pradyunsg) [inline pyce/_imports.py]: Don't have a mutable default.

https://docs.python-guide.org/writing/gotchas/

**Review by pradyunsg — CHANGES_REQUESTED** [review]

File: pyce/_imports.py (hunk 1 above)
Comment (theonewolf) [reply pyce/_imports.py]: Thanks @pradyunsg!  I seem to have regressed this morning.  Going to fix that after this cup of coffee.

I'm also adding type hints.

**Comment by pradyunsg** [comment]
You still have a mutable default (dictionaries are mutable). :/

**Comment by theonewolf** [comment]
@pradyunsg that should be resolved now.

**Comment by theonewolf** [comment]
This closed #22.

**Pull request merged**